src/main/java/com/cloudogu/smeagol/wiki/usecase/Commits.java

```diff
@@ -0,0 +1,20 @@
+package com.cloudogu.smeagol.wiki.usecase;
+
+import com.cloudogu.smeagol.Account;
+import com.cloudogu.smeagol.AccountService;
+import com.cloudogu.smeagol.wiki.domain.*;
+
+import java.time.Instant;
+
+public class Commits {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://ecosystem.cloudogu.com/sonar/coding_rules#rule_key=squid%3AS1118)